### ISSUE TITLE
Joomla Accessibility Team structure - proposed

### ISSUE THREAD

**Issue by zwiastunsw** [joomla/accessibility#24]
My proposal for the Joomla Accessibility Team structure
I think that in the future Joomla Accessibility Team should be composed of three subteams:
1. **Advisory Subteam**. 
**Task**: advising on accessibility, creating guides, helping in forums 
2. **Audit & Testing Subteam**. 
**Task**: Testing and evaluation of Joomla, Joomla elements, Joomla extensions, Joomla templates and Joomla.org websites
3. **Implementation Subteam**: 
**Task**: Design and development of accessibility improvements, extensions, etc.
4. **Promotion Subteam**:
**Task**: Promotion of JAT activities and initiatives, promotion accessibility issues among Joomla extensions developers, etc.

What do you think about this?
![jat_structure_proposed](https://user-images.githubusercontent.com/29527200/30192304-24fa643a-9447-11e7-8732-c606f0b91e10.png)

**Comment by armenos** [comment]
It’s a good to start with this discussion. Thank you Stefan for this nice JAT structure.

We need to think how we can be connected to other teams and making this visual in JAT structure. 
I think this link can inspire use to do so: 
http://w3c.github.io/wai-planning-and-implementation/Overview.html#resp

We need to insure that we can collaborate with other teams within Joomla Project. 
We need Liaison to other teams, this is important.

Needs to be reconsidered to be connected with teams which are working on this subject: 
UX team (good UX is good business), Marketing team, Content team, Project management, Testers and QA Teams, etc.

With this I’m trying to say, this structure is good. 
But we need to add Liaisons to other groups/teams

**Comment by zwiastunsw** [comment]
> We need to think how we can be connected to other teams and making this visual in JAT structure.

I fully agree!

**Comment by armenos** [comment]
By integrating this #20 discussion to this structure we will be clear with our goals and responsibility
Don’t you think?

**Comment by zwiastunsw** [comment]
Of course :)

**Comment by armenos** [comment]
I’m suggesting to make a new sub-team “Audit & Testing” for now with Stefan Wajda as sub-team leader. So he can manage his team members and be free to go forward.
Of course, if Stefan agrees on whit this.
